core/test/unit/config_spec.js

```diff
@@ -377,6 +377,11 @@ describe('Config', function () {
                 testData = {nav: {url: 'http://my-ghost-blog.com/blog/short-and-sweet/'}};
                 config.urlFor(testContext, testData).should.equal('http://my-ghost-blog.com/blog/short-and-sweet/');
 
+                configUtils.set({url: 'http://my-ghost-blog.com'});
+                testData = {nav: {url: 'http://my-ghost-blog.com:3000/short-and-sweet/'}};
+                config.urlFor(testContext, testData).should.equal('http://my-ghost-blog.com:3000/short-and-sweet/');
+                config.urlFor(testContext, testData).should.not.equal('http://my-ghost-blog.com/:3000/short-and-sweet/');
```

Comment:
you can keep this line, but this line is not really needed - you already assert the correct result in line `382`.